--- origsrc/zinnia-0.06/ruby/zinnia_wrap.cpp	2009-05-30 23:39:40.000000000 -0500
+++ src/zinnia-0.06/ruby/zinnia_wrap.cpp	2018-04-26 13:16:22.804254700 -0500
@@ -1396,7 +1396,7 @@ SWIG_Ruby_AppendOutput(VALUE target, VAL
 /* Error manipulation */
 
 #define SWIG_ErrorType(code)                            SWIG_Ruby_ErrorType(code)               
-#define SWIG_Error(code, msg)            		rb_raise(SWIG_Ruby_ErrorType(code), msg)
+#define SWIG_Error(code, msg)            		rb_raise(SWIG_Ruby_ErrorType(code), "%s", msg)
 #define SWIG_fail                        		goto fail				 
 
 
