--- origsrc/zinnia-0.06/perl/Makefile.PL	2008-09-13 11:59:36.000000000 -0500
+++ src/zinnia-0.06/perl/Makefile.PL	2011-10-03 14:12:40.155077400 -0500
@@ -3,8 +3,8 @@ WriteMakefile(
     'NAME'		=> 'zinnia',
     'CC'                => 'c++',
     'LD'                => 'c++',
-    'INC'               => '',
-    'LIBS'              => '-lzinnia',
+    'INC'               => '-I..',
+    'LIBS'              => '-L../.libs -lzinnia',
 #    'VERSION'	        => '0.1',
     'OBJECT'            => 'zinnia_wrap.o' 
 );
--- origsrc/zinnia-0.06/python/setup.py	2008-09-13 11:59:31.000000000 -0500
+++ src/zinnia-0.06/python/setup.py	2011-10-03 14:15:28.385770700 -0500
@@ -7,5 +7,7 @@ setup(name = "zinnia-python",
       py_modules=["zinnia"],
       ext_modules = [Extension("_zinnia",
                                ["zinnia_wrap.cxx",],
+                               include_dirs=['..'],
+                               library_dirs=['../.libs'],
                                libraries=["zinnia"])
                      ])
--- origsrc/zinnia-0.06/ruby/extconf.rb	2008-09-13 11:59:32.000000000 -0500
+++ src/zinnia-0.06/ruby/extconf.rb	2011-10-03 14:21:04.071955800 -0500
@@ -2,5 +2,6 @@ require 'mkmf'
 
 zinnia_config = with_config('zinnia-config', 'zinnia-config')
 use_zinnia_config = enable_config('zinnia-config')
-have_library("zinnia")
-have_header('zinnia.h') && create_makefile('zinnia')
+find_library("zinnia", 'main', '../.libs')
+have_library('stdc++')
+find_header('zinnia.h', '..') && create_makefile('zinnia')
